README.md

```diff
@@ -1,19 +1,31 @@
-# Primitives for Alpaka
-This contains a `reduce` and a `transform` primitive for Alpaka, both header-only.
-Directories:
-* `include/vikunja`: The actual code
-    - `mem/iterator`: Contains a base iterator class and a policy based iterator that can vary the memory access pattern: Either a linear or a grid-striding access is possible.
-    - `reduce`: Contains the `transform_reduce` and the `reduce` function (the latter is just a convenient wrapper around the former).
-        + `detail`: This contains the two possible reduce kernels.
-    - `transform`: Contains two `transform` variants with one and two input iterators.
-        + `detail`: This contains the transform kernel.
-    - `workdiv`: This contains various working divisions for the different backends.
+# Vikunja
+
+[![Build Status](https://gitlab.com/hzdr/crp/vikunja/badges/master/pipeline.svg)](https://gitlab.com/hzdr/crp/vikunja/-/commits/master/)
+[![Documentation Status](https://readthedocs.org/projects/vikunja/badge/?version=latest)](https://vikunja.readthedocs.io)
+[![Doxygen](https://img.shields.io/badge/API-Doxygen-blue.svg)](https://vikunja.readthedocs.io/en/latest/doxygen/index.html)
+[![Language](https://img.shields.io/badge/language-C%2B%2B14-orange.svg)](https://isocpp.org/)
+[![Platforms](https://img.shields.io/badge/platform-linux-lightgrey.svg)](https://github.com/alpaka-group/vikunja)
+[![License](https://img.shields.io/badge/license-MPL--2.0-blue.svg)](https://www.mozilla.org/en-US/MPL/2.0/)
+
+![vikunja](docs/source/logo/vikunja_logo.png)
+
+Vikunja is a performance portable algorithms library defines functions for a variety of purposes that operate on ranges of elements. It supports the execution on multi core CPUs and various GPUs.
+
+Vikunja uses [alpaka](https://github.com/alpaka-group/alpaka) to implement platform independent primitives such like `reduce` or `transform`. 
 
 # Installation
+## Install Alpaka
 
-## Requirements
+Alpaka requires a [boost installation](https://github.com/alpaka-group/alpaka#dependencies).
 
-[Alpaka 0.6.0](https://github.com/alpaka-group/alpaka)
+```bash
+git clone --depth 1 --branch 0.8.0 https://github.com/alpaka-group/alpaka.git
+cd vikunja/build
+cmake ..
+cmake --install .
```

Comment:
Is it really sufficient to just clone alpaka next to the vikunja repo? For LLAMA, I require that alpaka is also installed.

Reply:
The code clones alpaka without installing alpaka. Then you go into a directory in your vikunja clone and configure CMake there. How can vikunja find alpaka, if it is not installed? The call to `cmake ..` should fail, because it cannot find alpaka.

Reply:
Ah, I see the problem. It is a copy and past error. The `cd` command is wrong. It should be `cd alpaka/build`.